src/components/RegionMap/RegionMap.tsx

```diff
@@ -143,36 +148,22 @@ function getProjectionConfig(
   return { scale, center };
 }
 
-function buildCountyGeometries(countyFipsList: string[]) {
-  const objects = COUNTIES_JSON.objects;
-  const geometries = objects.counties.geometries.filter(geoCounty =>
-    countyFipsList.includes(geoCounty.id),
+function buildCountyGeometries(
+  countiesTopoJson: any,
```

Comment:
Can this be a type?

Reply:
That somehow breaks the calls to `geoBounds` and `geoCentroid`, I'm not sure how to make those compatible 🤷‍♂️

Reply:
I think TypeScript's `typeof import()` syntax helps here.  See https://github.com/covid-projections/covid-projections/pull/3088

We could probably use that to remove some of our existing hand-defined types too...